examples/n2hp_cube_example.py

```diff
@@ -32,6 +33,8 @@
 else:
     # Run the fitter
     # Estimated time to completion ~ 2 minutes
+    spc.xarr.refX = 23*u.Hz
+    spc.xarr.velocity_convention = 'radio'
```

Comment:
I added these two lines to get fiteach to work (it was failing to convert to Hz from m/s) but I set the refX to 23 Hz completely randomly.